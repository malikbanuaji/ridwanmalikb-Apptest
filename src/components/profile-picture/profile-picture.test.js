import * as React from 'react';
import ProfilePicture from './profile-picture';

import {render} from '@testing-library/react-native';

test('should render properly', () => {
  const component = render(<ProfilePicture photoUri="file://uri" />);
  expect(component.toJSON()).toMatchSnapshot();
});

test('should render properly without photos', () => {
  const component = render(<ProfilePicture photoUri="N/A" />);
  expect(component.toJSON()).toMatchSnapshot();
});
