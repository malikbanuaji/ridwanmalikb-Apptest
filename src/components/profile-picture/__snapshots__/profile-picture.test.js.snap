// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render properly 1`] = `
<Image
  source={
    Object {
      "uri": "file://uri",
    }
  }
  style={
    Object {
      "borderRadius": 8,
      "height": 42,
      "width": 42,
    }
  }
/>
`;

exports[`should render properly without photos 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#99aab5",
      "borderRadius": 8,
      "height": 42,
      "width": 42,
    }
  }
/>
`;
